instances/dev.tf

```diff
@@ -46,14 +46,10 @@ data "aws_security_group" "dev_sg" {
   name = "Development"
```

Comment:
> the Development security group has now been modified to whitelist everyone on port 22

Just to double-check my understanding: the `dev_sg` security group is not managed by terraform, and so that modification was made in the AWS web UI and is not tracked by this repo; is that correct?

Reply:
Terraform will only manage EC2 instances, AMIs, and (soon) load balancers and autoscaling groups (used by future staging + prod).